lib/index.js

```diff
@@ -94,14 +100,30 @@ async function initDatabase (server, dbOptions, schema, accounts) {
   schema = schema.replace(/\$\$/g, ';').replace(/DELIMITER\s+;/gi, '');
   await Common.doQuery(schema, null, connection);
 
-  const accountList = (accounts || '').split(',').map((account) => {
-    return account.trim();
-  }).filter((account) => { return !!account; });
+  const toAccountList = function (input) {
```

Comment:
This `map()`-`filter()` combo could be a `reduce()`. No pressure though. I hate `reduce()` and this is startup code.